hack/operatorhub/cmd/buildkite/buildkite.go

```diff
@@ -0,0 +1,207 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package buildkite
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/cloud-on-k8s/v2/hack/operatorhub/cmd/flags"
+)
+
+const (
+	url = "https://api.buildkite.com/v2/organizations/elastic/pipelines/cloud-on-k8s-operator/builds"
+)
+
+// Command will return the buildkite command
+func Command() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "buildkite",
+		Short:        "Start operatorhub release operation within Buildkite",
+		Long:         "Start operatorhub release operation within Buildkite.",
```

Comment:
I think
```bash
curl -XPOST "https://api.buildkite.com/v2/organizations/elastic/pipelines/cloud-on-k8s-operator-release-operatorhub/builds" \
-H "Authorization: Bearer $BK_TOKEN" \
-d '{
    "commit": "2.6.1",
    "branch": "2.6"
}'
```
could have been enough.

Reply:
We discussed this privately, and decided that opening #6408 would be beneficial.